scripts/ast-create-node.ts

```diff
@@ -21,6 +29,36 @@ export async function main(
 		return 1;
 	}
 
+	if (!(await languageExists(language))) {
+		const languages = await getLanguages();
+
+		reporter.error(
+			markup`Language argument "${language}" is not a valid language`,
+		);
+
+		reporter.info(markup`The following languages are valid:`);
+
+		languages.forEach((languageName) => {
+			reporter.log(markup`${languageName}`);
+		});
+		return 1;
+	}
+
+	if (!(await languageCategoryExists(language, category))) {
+		const categories = await getLanguageCategories(language);
+
+		reporter.error(
+			markup`Category argument "${category}" is not a valid category for "${language}"`,
+		);
+
+		reporter.info(markup`The following categories are valid for ${language}:`);
+
+		categories.forEach((categoryName) => {
+			reporter.log(markup`${categoryName}`);
+		});
```

Comment:
```suggestion
		reporter.list(categories.map((categoryName) => markup`${categoryName}`);
```

can be used to output as a nicely formatted list.

Reply:
@sebmck I believe some checks failed because of your new code: `reporter.list(categories.map((categoryName) => markup`${categoryName}`);` should be `reporter.list(categories.map((categoryName) => markup`${categoryName}`));` instead

Reply:
#9422b50a38378796d2d